### PR TITLE
dvc 2.2.0

### PR DESCRIPTION
**Pull request by efiop**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by carlocab — DISMISSED** [review]
Thanks, @efiop.

**Comment by efiop** [comment]
Hm, interesting, looks like `resources` don't include `s3fs` that we've started using in 2.2.0 . Please do not merge, I'm investigating...

File: Formula/dvc.rb [hunk 1]
```diff
@@ -619,6 +619,7 @@ def install
   end
 
   test do
-    system "#{bin}/dvc", "version"
+    output = pipe_output("#{bin}/dvc doctor 2>&1", nil, 0)
```
Comment (carlocab) [inline Formula/dvc.rb:622]:
```suggestion
    output = shell_output("#{bin}/dvc doctor 2>&1")
```

Comment (efiop) [reply Formula/dvc.rb]: Thank you! :pray:

**Comment by carlocab** [comment]
```
  Minitest::Assertion: Expected /Supports:\ All\ remotes/ to match "DVC version: 2.2.0 (brew)\n---------------------------------\nPlatform: Python 3.9.5 on macOS-11.4-arm64-arm-64bit\nSupports: azure, gdrive, gs, http, https, ssh, oss, webdav, webdavs\n".
```
Maybe you need to do `brew update-python-resources` again?

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**